src/main/java/net/glowstone/GlowWorld.java

```diff
@@ -1714,23 +1806,25 @@ public boolean createExplosion(double x, double y, double z, float power, boolea
     }
 
     @Override
-    public boolean createExplosion(double x, double y, double z, float power, boolean setFire, boolean breakBlocks) {
+    public boolean createExplosion(double x, double y, double z, float power, boolean setFire,
+            boolean breakBlocks) {
         return createExplosion(null, x, y, z, power, setFire, breakBlocks);
     }
 
     /**
      * Create an explosion with a specific entity as the source.
      *
-     * @param source      The entity to treat as the source, or null.
```

Comment:
For me, IntelliJ will align the description like that automatically. Is that against our new checkstyle?

Reply:
No. Reverted; for me, IntelliJ removes that kind of alignment while reformatting the code to wrap long lines (which I do on one file at a time when there are too many to do manually).

Reply:
This change is present in other files in this PR, you might want to check.

Reply:
Reverted throughout GlowWorld; could find no instances involving existing Javadocs in other files.